lib/tools/settings-client-commands.js

```diff
@@ -171,6 +171,7 @@ commands.setDeviceSysLocaleViaSettingApp = async function setDeviceSysLocaleViaS
  * @property {number|string} longitude - Valid longitude value.
  * @property {number|string} latitude - Valid latitude value.
  * @property {?number|string} altitude - Valid altitude value.
+ * @property {?number|string} speed - Valid speed value. Should be bigger then 0.0 meters/second.
```

Comment:
Should be greater than ...

Reply:
could it be zero?

Reply:
true story! 🤦 😄

Reply:
Should be strictly grater then 0.0 can be a very very small number but must be larger then 0.0 since 0.0 is considered to be invalid on mobile android platform.